app/pods/attempt/template.hbs

```diff
@@ -4,11 +4,11 @@
             <div class="button-dashed" {{action 'toggleSideBar'}}>
               <button>
                 {{#if (not sidebarCollapsed)}}
-                  <i class="far fa-times-circle"></i>
+                  <i class="far fa-times-circle"></i>&nbsp;
                 {{else}}
-                  <i class="fas fa-bars"></i>
+                  <i class="fas fa-bars"></i>&nbsp;
```

Comment:
Please use padding or margin, using &nbsp is not considered a good practice to follow

Reply:
Shall i push the CSS changes here in addition to motley? @witty123

Reply:
@witty123 @abhishek97 these PRs are on hold right now because i don't know whether i should push the CSS changes to motley only or should I push them to both motley and this repo. Should I revert the CSS changes i made in the PRs in this repo, or should I leave them be and just modify them?